packages/mobile/src/app/selectors.ts

```diff
@@ -19,3 +23,13 @@ export const getLastTimeBackgrounded = (state: RootState) => {
 export const sessionIdSelector = (state: RootState) => {
   return state.app.sessionId
 }
+
+export const verificationPossibleSelector = (state: RootState) => {
+  const e164Number = e164NumberSelector(state)
+  const dollarBalance = stableTokenBalanceSelector(state)
+  const saltCache = e164NumberToSaltSelector(state)
+  return !!(
+    (e164Number && saltCache[e164Number]) ||
+    (dollarBalance && isSufficientBalanceForQuotaRetrieval(dollarBalance))
+  )
+}
```

Comment:
great!
Wondering, should we also include the check for `!state.app.numberVerified` in there?

Reply:
I think we should keep this separate, cause they mean different things - "is it possible?" and "was it done already?". WDYT?